packages/integration-tests/api/collections.js

```diff
@@ -16,7 +16,7 @@ const collectionsApi = require('@cumulus/api-client/collections');
  */
 const createCollection = (params) => {
   deprecate('@cumulus/integration-tests/collections.createCollection', '1.21.0', '@cumulus/api-client/collections.createCollection');
-  return collectionsApi.getGranule(params);
+  return collectionsApi.createCollection(params);
```

Comment:
..... wow.        How was that not causing failures all over??

That's astonishing.  :(

Reply:
Hmm, doing a quick audit of the code base.... it looks like this isn't referenced.  We should just remove it, although I'm fine with relegating that to 'not here'.      I am adding tearing this all out to the IP sprint list.

Reply:
Yeah I just noticed it and figured I would fix it